src/DynamoCore/Models/NodeModel.cs

```diff
@@ -1737,6 +1728,11 @@ private void OnRenderPackageUpdateCompleted(AsyncTask asyncTask)
                 renderPackages.Clear();
                 renderPackages.AddRange(task.RenderPackages);
                 HasRenderPackages = renderPackages.Any();
+
+                if (RenderPackages.Any())
+                {
+                    OnUpdatedRenderPackagesAvailable(task.RenderPackages);
+                }
```

Comment:
Can we remove the `RenderPackages` field from `NodeModel`? It seems unnecessary, no?